classes/stats_generator.rb

```diff
@@ -40,7 +41,8 @@ def upload
       vputs data.to_s
 
       # send it to the db
-      handle_request REST.post("http://cocoadocs-api.cocoapods.org/pods/#{spec.name}", data.to_json)
+      api = ENV["COCOADOCS_API"] || 'http://cocoadocs-api.cocoapods.org'
+      handle_request REST.post("/pods/#{spec.name}", data.to_json)
```

Comment:
https://github.com/CocoaPods/cocoadocs.org/commit/5381c1c2112a9207b0a62b7a35adacdd18f85d95

Shame, was got caught by the CI there too,